src/redturtle/prenotazioni/content/prenotazione.py

```diff
@@ -174,6 +173,13 @@ class IPrenotazione(model.Schema):
         title=_("Expiration date booking"), required=True
     )
 
+    directives.mode(booking_code="display")
+    booking_code = schema.TextLine(
+        title=_("Booking code"),
+        description=_("Codice univoco della prenotazione"),
```

Comment:
già che ci sei, uniformiamo le label in questo form? C'è un misto tra inglese/italiano che mi da fastidio ;)

Reply:
@cekk aggiornato:

     booking_code = schema.TextLine(
         title=_("label_booking_code", default="Booking code"),
         description=_("description_booking_code", default="Unique booking code"),
         required=False,
     )